draft-ietf-rats-endorsements.md

```diff
@@ -201,13 +201,33 @@ some minimal matching policy (e.g., exact match against a singleton reference
 value).  This unfortunately complicates design as a Verifier may need
 multiple parsers for matching policies.
 
-# Endorsing Identity
+# Key Material for Attestation
 
-One type of claims that might be endorsed would be claims having to do with
-identity, such as verification keys bound to an identifier for the Attesting Environment (e.g., {{-rats-eat}} `ueid`).  While identity claims are just another
+A Verifier receives Evidence from an Attester that must be considered untrusted
+until established otherwise. This trust is established using cryptography. Typically,
+the bottom-most Attesting Environment in an Attester will sign claims about a Target Environment
+with a private key that that Attesting Environment possesses and the Verifier will verify
+them with a public key it possesses, called a verification key below. While this is typical,
+cryptography other than public key may also be used.
+
+Since it is not possible to establish trust in an Attester without cryptography,
+the Verifier must have a verification key for each Attester. Typically, the verification
```

Comment:
The sentence _"[t]he Verifier must have a verification key for each Attester."_ may give the impression that a Verifier must be provisioned with the key that verifies the Attester's signature.  In reality, the requirement is less stringent: it is sufficient that the Verifier is given enough information to trust the verification key (e.g., via one or more CA certs) rather than to _"have"_ such verification key.

Reply:
Yes. Very much agreed.

Would be good to add text explaining this -- that an endorsement might contain a root certificate or other key material that is needed to establish trust in the attester 

(and it would help get closer to my prediction that 3 pages were needed :-) )

Reply:
> Updated text. Is the new text better?

WFM, thanks!